force-app/main/default/objectTranslations/SMS__c-no/SMS__c-no.objectTranslation-meta.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Denne fila ligger i crm-platform-integration, og kan derfor ikke ligge i vår pakke.

Reply:
> Kanskje dette egentlig ikke er HOT-spesifikk funksjonalitet, og at dette skal ligge i platform-integration? @chfosli

Jeg tenker den bør ligge i crm-hot, så har vi full kontroll på hvordan det funker og trenger ikke ta hensyn til andre team.

Reply:
Den er grei. Må denne oversettelses-fila flyttes til unpackaged, da? Siden den inneholder oversettelse av flowen (HOT_ProsessedAction)?

Reply:
Ja blir vel for det da. Ser vi har lagt oversettelsen for en del felles objekter her: https://github.com/navikt/crm-platform-unpackaged/tree/master/force-app/main/default/objectTranslations.

Vi kan ha den filen flere steder, må vel faktisk ha den der du har oversett felterene tror jeg, men kan være tom, eller bare inneholde de elementene som er oversatt i crm-hot.

Se f.eks. https://github.com/navikt/crm-hot/blob/master/force-app/main/default/objectTranslations/ServiceAppointment-no/ServiceAppointment-no.objectTranslation-meta.xml og https://github.com/navikt/crm-platform-unpackaged/blob/master/force-app/main/default/objectTranslations/ServiceAppointment-no/ServiceAppointment-no.objectTranslation-meta.xml

Reply:
Så konklusjonen her er? :s

Reply:
Konklusjonen er: 

- Fila skal ligge i 2 repositories: crm-hot og crm-platform-unpackaged
- crm-hot: Fila skal være tom (utenom xml-tag i toppen)
- Unpackaged: Fila med innhold (oversettelser)